components/lusha/actions/contact-enrich/contact-enrich.mjs

```diff
@@ -0,0 +1,37 @@
+import lusha from "../../lusha.app.mjs";
+
+export default {
+  key: "lusha-contact-enrich",
+  name: "Enrich Contacts",
+  description: "Enriches contacts based on provided IDs. [See the documentation](https://www.lusha.com/docs/#contact-enrich)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    lusha,
+    requestId: {
+      propDefinition: [
+        lusha,
+        "requestId",
+      ],
+      label: "Company Request ID",
+      description: "The request ID generated from the company search response.",
+    },
+    contactIds: {
+      propDefinition: [
+        lusha,
+        "contactIds",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const response = await this.lusha.enrichContacts({
+      $,
+      params: {
+        requestId: this.requestId,
+        contactIds: this.contactIds,
+      },
```

Comment:
Wouldn't it be data since it's a POST request?
```suggestion
      data: {
        requestId: this.requestId,
        contactIds: this.contactIds,
      },
```